library/src/include/transform.h

```diff
@@ -6,13 +6,15 @@
 #ifndef TRANSFORM_H
 #define TRANSFORM_H
 
+#include "rocfft_hip.h"
+
 struct rocfft_execution_info_t
 {
-	void *workBuffer;
-	size_t workBufferSize;
-
-	rocfft_execution_info_t() : workBuffer(nullptr), workBufferSize(0)
-	{}
+    void *workBuffer;
+    size_t workBufferSize;
+    hipStream_t rocfft_stream = 0;//by default it is stream 0
```

Comment:
default must be initialized in the constructor initializer list

Reply:
hmm, i see c++ 11 allows this, never mind

Reply:
Yes, the rocBLAS also use this feature. 
https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/library/src/include/handle.h#L31